src/osyris/plot/wrappers.py

```diff
@@ -126,3 +126,50 @@ def scatter(ax, x, y, data, **kwargs):
         return coll
     else:
         return ax.scatter(x, y, **default_args)
+
+def line_integral_convolution(ax, x, y, z, **kwargs):
+    """
+    Wrapper that plots a line integral convolution of a vector field.
+    Uses alpha blending to merge the LIC with a user-requested color.
+    """
+    import lic as lic
+    from matplotlib.cm import ScalarMappable
+    from matplotlib.colors import LogNorm
+
+    #compute line integral convolution
+    try:
+        lic_res = lic.lic(z[...,1], z[...,0], length=kwargs["length"])
+        del kwargs["length"]
+    except KeyError:
+        lic_res = lic.lic(z[...,1], z[...,0], length=30)
+
+    plot_args = {**kwargs}
+    plot_args["extent"] = [np.min(x),np.max(x),np.min(y),np.max(y)]
```

Comment:
Note that here the `x` and `y` are pixel centers, and we actually need the pixel edges for the true extent.

Reply:
The to_bin_edges function in tools.py doesn't work; it's trying to concatenate arrays with simple floats. I think this is an adequate fix: 
```
def to_bin_edges(x):
    """
    Convert array centers to edges
    """
    centers = to_bin_centers(x)
    dx = x[1]-x[0]
    return np.append(centers-dx,np.array([centers[-1],centers[-1]+dx]))
```